tests/cli/ReadersTest.cpp

```diff
@@ -0,0 +1,58 @@
+//
+// Created by Illia Plaksa on 19.01.2022.
+//
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "cli/include/Readers.h"
+#include "mocks/ConsoleManipulatorMock.h"
+
+class ReadersTest : ::testing::Test
+{
+};
+
+TEST(ReadersTest, shouldReadLabels)
+{
+    auto ostream = std::stringstream{};
```

Comment:
code style suggestion: why not 
std::stringstream ostream;
? (vladimir-moiseiev mentioned it once)

Reply:
Fixed